schemas/pioneer-core/demographic-survey/demographic-survey.1.schema.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$comment": "mozilla-ion/ion-core-addon#182 - Collect demographic information",
+  "$comment": "https://github.com/mozilla-rally/rally-core-addon/pull/624 - new exactIncome field for Rally Core Add-On demographic survey",
```

Comment:
Perhaps put this as a comment/description of the new field, leaving the top-level one for reference of when the entire ping was added.